core/domain/state_domain.py

```diff
@@ -2116,10 +2113,11 @@ def get_image_ids_of_state(self):
         """
         content_html = self.content.html
         interaction_html_list = self.interaction.get_all_html_content_strings()
-        if interaction_html_list != []:
-            state_html = content_html + interaction_html_list[0]
-        else:
-            state_html = content_html
+
+        state_html = content_html
+        for html in interaction_html_list:
```

Comment:
Does this fix the issue you mentioned last meeting?

Reply:
yup. It got fixed.